### DISCUSSION TITLE
📣 GitHub wants to know how we can make automations in Projects better for you

### DISCUSSION THREAD

**Discussion by jaredsb** [community/community#15157] [Projects and Issues] [answered]
Hi Everyone 👋🏼 , TheProjects Beta team here at GitHub is working to improve our automations in planning and tracking!  If you’re interested in telling us how we can make automations more useful for you please take 👉🏼 [this survey](https://github.surveymonkey.com/r/HLHXXWZ) 👈🏼

It takes about 10 minutes to complete, but to thank you for your time we will randomly select 5 respondents for a $100 worth of swag from the GitHub store.  The survey will be open until May 2nd.

**Comment by Apriadi000** [comment]
Kenapa github tidak bisa di hubungkan ke herkoku

**Reply by STOCKHAMMER-4663-RAHSTORF-21** [reply]
Hi ! This is a Test ! Lg

**Reply by KINGM232** [reply]
dds

**Reply by chukwujustin** [reply]
You should consider adding delivery timelines.

**Reply by 8mahfouz** [reply]
صعبة ولكن يمكن مع الوقت والتكرار لابد 😀مع تكرار الفشل سوف تفهم  اتفاجئ اني بدئت اتفهم👏👀

**Comment by vedantmgoyal9** [comment]
I just filled out the survey. Eagerly waiting for these automation since they'll be very helpful and save maintainers' time in managing repos (huge repos like VSCode).

**Comment by Andre601** [comment]
Is it me or does the survey primarely focus on active, full-time devs/orgs and not also community-driven open source projects?
Would like to participate in the survey, but the second question alone already gives me the impression that I as a free-time contributor am not the target demographic of the survey...

**Reply by jaredsb** [reply]
Hi @Andre601, we are definitely interested in how automations can help open source maintainers, so I'd love to know what you think. What about the question made it seem like that wasn't the case?

**Reply by Andre601** [reply]
The second question. It basically assumes I and others are "full-time developers" which certainly isn't the case for projects that are maintained in the free time of people.

![image](https://user-images.githubusercontent.com/11576465/164107824-bc24ab85-9013-4583-b277-55a37caefb6c.png)

That's why I assume the survey is more for companies or groups with full-time devs working on projects.

**Reply by jaredsb** [reply]
Got it, that makes sense. Thanks for calling that out!

**Reply by Andre601** [reply]
Also, first question confuses me about a few things... This is primarily because my native language isn't english, so I can't quite understand what the purpose of a DevOps is and what the differences between a "Project Manager" and a "Product Manager" are.

**Reply by jaredsb** [reply]
DevOps engineers are people that work on the developer operations at an organization. It's basically a specialized engineering role. Project and Product managers are different stakeholders that work with engineering teams to plan and track work.

**Comment by AnatoliiKlesh** [comment]
Done.
What next? :)

**Comment by RubenKelevra** [comment]
Just used the classic projects feature. In general it would be neat to be able to have some kind of scripting based on tags.

I looked at the projects beta and it's kind of much to learn "just for fun" while it might get trashed and replaced in the future with something different. So I'll wait for this to be stable.

**Comment by RubenKelevra** [comment]
I always run in the issue that people need to provide more feedback. I would love to mark them by name in the metadata of the ticket/PR and set a label for "blocked/need more info" or "blocked/need rework" etc. and remove it automatically when they respond/push a new commit.

Rechecking "blocked/need more info"-tags is a major efficiency issue with the github issues/prs.

**Comment by ian-steffy** [comment]
Every github automation action provided by github, like the [alex-page/github-project-automation-plus](https://github.com/alex-page/github-project-automation-plus), or from anyone else is broken on Github Projects Beta if it requires one to move a card or use a project URL. 
Your automation tools leave out the incredible important option to move a card that is in Review to a Review column.


One suggestion is that you tell users this before they choose Github Project (beta) over Github Projects.

**Comment by josemlbq** [comment]
Wouldn't it be nice if we could assign _"story points"_ to issues so that we can have agile metrics in projects? 😍

**Comment by jaredsb** [comment, marked as answer]
Thanks to everyone for their input. I'll be reaching out to winners this week with your gift cards!

**Reply by MikeArep** [reply]
TQ so much..
